src/test/kotlin/lotto/domain/LottosTest.kt

```diff
@@ -0,0 +1,32 @@
+package lotto.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+
+class LottosTest {
+    @Test
+    fun `로또 리스트 - 리스트 반환 테스트`() {
+        // given
+        val lotto = Lotto(listOf(LottoNumber.from(1), LottoNumber.from(2), LottoNumber.from(3), LottoNumber.from(4), LottoNumber.from(5), LottoNumber.from(6)))
```

Comment:
테스트 코드도 중복을 의식하면서 리팩토링 해주면 좋을 거 같은데요~
다음과 같이 간결하게 바꿔줄 수도 있겠네요!

```suggestion
        val lotto = Lotto(listOf(1, 2, 3, 4, 5, 6).map { LottoNumber.from(it) })
```

Reply:
오... 좋은 팁 감사합니다! 🙏 테스트 코드 작성 중 `Lotto` 객체 생성시에 복수의 `LottoNumber` 객체를 생성하는 부분의 코드가 지저분해 마음이 쓰였는데요. 리뷰어님 코멘트를 보니 이 부분이 정리가 되었습니다! 😃

- https://github.com/next-step/kotlin-lotto/commit/5763d8a2b29f92e048e9b913030bf3ee86e53dd2 , https://github.com/next-step/kotlin-lotto/commit/4b355df2b00a43f5bd6b45bed1158b1e7f98d71c

추후 도전 과제를 진행하면서도 이 부분 꼭 기억해 두겠습니다! 감사합니다 🙏